app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -0,0 +1,29 @@
+@import "twitter-bootstrap-static/bootstrap";
+@import "twitter-bootstrap-static/fontawesome";
+
+body {
+  background: #f3f4f5;
+}
+
+// Make navigation & page header background white
+.navbar-default,
+._page-header {
+  background-color: #FFF;
+}
+
+.navbar {
+  margin-bottom: 0;
+}
+
+._page-header {
+  border-bottom: 1px solid #e7e7e7;
```

Comment:
Color literals like `#e7e7e7` should only be used in variable declarations; they should be referred to via variable everywhere else.